Src/DependencyCollector/Shared/Implementation/DesktopDiagnosticSourceHttpProcessing.cs

```diff
@@ -0,0 +1,81 @@
+﻿#if NET45
+namespace Microsoft.ApplicationInsights.DependencyCollector.Implementation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.DependencyCollector.Implementation.Operation;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.ApplicationInsights.Web.Implementation;
+
+    /// <summary>
+    /// Concrete class with all processing logic to generate RDD data from the callbacks received from HttpDesktopDiagnosticSourceListener.
+    /// </summary>
+    internal sealed class DesktopDiagnosticSourceHttpProcessing : HttpProcessing
```

Comment:
Could you please show me where FrameworkEventSource reports failure cases?
I don't see it here [here](https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/0e5ac6e38b2e842d6b132beffbced90dc484ae73/Src/DependencyCollector/Shared/Implementation/FrameworkHttpEventListener.cs)

And in case of HTTP failure means exception because of e.g. DNS or network issue or cancellation.